### DISCUSSION TITLE
Renaming indexes

### DISCUSSION THREAD

**Discussion by TrueBram** [ravendb/ravendb#14235] [Q&A] [answered]
Hi,

I recently saw in the documentation that you can subscribe to an index rename change. When is this change triggered? I couldn't find anything about renaming indexes in the documentation.

https://ravendb.net/docs/article-page/5.3/csharp/client-api/changes/how-to-subscribe-to-index-changes

**Comment by garayx** [comment, marked as answer]
Hi 

There is no option to rename the indexes on RavenDB 4.x/5.x, therefor no option to subscribe to such change. We will fix the docs asap
